tests/neg-macros/newClassParamsMissingArgument/Macro_1.scala

```diff
@@ -0,0 +1,24 @@
+//> using options -experimental
+
+import scala.quoted._
+
+inline def makeClass(inline name: String): Object = ${ makeClassExpr('name) }
+private def makeClassExpr(nameExpr: Expr[String])(using Quotes): Expr[Object] = {
+  import quotes.reflect.*
+
+  val name = nameExpr.valueOrAbort
+  val parents = List(TypeTree.of[Object])
```

Comment:
there should be a test case where the parent class has a constructor argument supplied via the constructor parameters

Reply:
I tried doing that in the not-very-well-named tests/run-macros/newClassParamsExtendsClassParams, which generates something like this:
```scala
'{
   class `name`(idx: Int) extends Foo(idx)
   new `name`(22)
}
```
Or do you mean something else?

Reply:
yeah that looks good